kickstarter/ios-oss
# Fix architecture warnings when targetting the simulator

# 📲 What

Fix architecture warnings when targeting the simulator.

# 🤔 Why

When we originally removed support for 32-bit applications, we changed the value of `Valid Architectures` to `arm64 arm64e`. This wasn't strictly necessary because the `Valid Architectures` setting is used if you want to specify different architectures for each target. According to the [Xcode docs](https://help.apple.com/xcode/mac/10.1/#/itcaec37c2a6?sub=devf0a9d5aca), the`Valid Architectures` list is intersected with the `Architectures` list and the resulting list is built. Because the `Architectures` list now specifies `ARCHS_STANDARD_64_BIT`, its not necessary to modify the `Valid Architectures` list in order to exclude 32-bit architectures. Resetting the `Valid Architectures` setting to the default fixed the warning. The general consensus on [stack overflow](https://stackoverflow.com/questions/12701188/whats-the-difference-between-architectures-and-valid-architectures-in-xcode) seems to be that you use the `Architectures` build setting to set the architectures you want to build and you leave `Valid Architectures` alone.

# 👀 See

**Before**
![image](https://user-images.githubusercontent.com/3156796/51572303-f9596080-1e72-11e9-9bae-967139e63c9e.png)

**After**
<img width="424" alt="screen shot 2019-01-22 at 5 34 15 pm" src="https://user-images.githubusercontent.com/3156796/51572464-82709780-1e73-11e9-81ae-c1f85086a579.png">


# How to test
- Checkout branch and build targetting the simulator. There should be no architecture warnings on the left panel.
